src/login/translation/languages/sv.json

```diff
@@ -187,6 +189,10 @@
   "mfa.two-factor-authn": "Tvåfaktorsautentisering",
   "mobile.button_add": "Lägg till",
   "mobile.confirm_mobile_placeholder": "Bekräftelsekod",
+  "mobile.confirm_code_wrong_length": "Koden måste vara 10-12 tecken lång, endast bokstäver och nummer tillåtna",
+  "security.confirm_security_length": "Max längden av 50 tecken har överskridits",
+  "security.confirm_security_placeholder": "Beskriv din säkerhetsnyckel",
+  "security.help_text": "max längd av 50 tecken" ,
```

Comment:
- "maxlängd på 50 tecken" - but have you considered the shorter: "max 50 tecken"?
- help text not translating to Swe: ![Screenshot 2020-09-01 at 18 04 12](https://user-images.githubusercontent.com/30963614/91877039-c184e280-ec7d-11ea-9b5b-88a37c0e0ee1.png)